### DISCUSSION TITLE
Insufficient privileges to complete the operation when using $filter parameter

### DISCUSSION THREAD

**Discussion by averycd** [YannickRe/spfx-msgraph-peoplesearch#34] [Q&A] [answered]
Great webpart! Thanks for putting this together.
I'm trying to filter on **accountEnabled eq true** and I get an error: 

Insufficient privileges to complete the operation.

I am able to filter on simple things like **mail eq 'user@domain.com'** without error.

Any ideas? I am really hopeful to be able to only show enabled accounts.

Thank you

**Comment by YannickRe** [comment, marked as answer]
Did you add the additional permissions as described in the setup guide?
=> https://github.com/YannickRe/spfx-msgraph-peoplesearch#cli-for-microsoft-365

**Reply by averycd** [reply]
I think it's working now. I did run the Grant-PnPTenantServicePrincipalPermission -Scope "User.Read.All" -Resource "Microsoft Graph" command successfully but I was receiving the insufficient permissions error even after that. After waiting overnight, it works this morning. Do you know if there is a way to make the change immediate? I have to apply this in another tenant. Thanks again!

**Reply by YannickRe** [reply]
The change should be immediate, at least it used to be... You could try a sign out/sign in to refresh your token.

**Reply by averycd** [reply]
Thanks! That worked!

**Reply by averycd** [reply]
Do you know if there is a way to use the $filter parameter to only show people who are members of a specific AD group?

**Reply by YannickRe** [reply]
I'm sorry, I have no idea if that type of filter is supported...

**Comment by averycd** [comment]
Okay, I will let you know if I figure it out. Thanks.
